docs/concepts/cluster-administration/cloud-providers.md

```diff
@@ -56,6 +56,38 @@ Different settings can be applied to a load balancer service in AWS using _annot
 
 The information for the annotations for AWS is taken from the comments on [aws.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/aws/aws.go)
 
+# OpenStack
+This section describes all the possible configurations which can
```

Comment:
I would say there are more configuration parameters to add, for volume, for lb and for authentication.

https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/openstack/openstack.go

Reply:
Hey Spyros! Yeah there are, but we wanted to keep this PR small since it was a first contribution. Feel free to open an issue or submit a PR if you'd like to add the rest

Reply:
ok, thanks